locale/en_US/manager.po

```diff
@@ -1138,6 +1138,12 @@ msgstr "{$locale} locale reloaded for {$contextName}."
 msgid "notification.localeSettingsSaved"
 msgstr "Locale settings saved."
 
+msgid "notification.localeSettingsCannotBeSaved.emptyFormsOption"
+msgstr "Locale settings cannot be saved. Select at least one language at Forms"
```

Comment:
Change this to: "The language setting could not be saved. At least one language must be enabled for each option."

Then you can reuse that locale key for both cases.